include/seqan3/core/detail/pack_algorithm.hpp

```diff
@@ -6,241 +6,15 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
- * \brief Provides algorithms for meta programming, parameter packs and seqan3::type_list.
+ * \brief [DEPRECATED] Provides algorithms for meta programming, parameter packs and seqan3::type_list.
  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \deprecated This header is deprecated and will be removed in SeqAn-3.1.0;. Please \#include
+ *             <seqan3/utility/detail/pack_algorithm.hpp> instead.
  */
 
 #pragma once
 
-#include <utility>
+#include <seqan3/utility/detail/pack_algorithm.hpp>
```

Comment:
Didn't we want to think about if pack algorithms should go into the "type_pack" utility submodule? I wonder if we should do all the renaming work twice or maybe postpone this one until we decide where it finally goes?

(Note: These changes clash with Renes PR about the type trait splitting)

Reply:
Then I'll just remove the commit for the pack_algorithm